tests/unit/dataactvalidator/test_b18_object_class_program_activity_1.py

```diff
@@ -0,0 +1,45 @@
+from factory import fuzzy
+from tests.unit.dataactcore.factories.staging import ObjectClassProgramActivityFactory
+from tests.unit.dataactvalidator.utils import number_of_errors, query_columns
+
+
+_FILE = 'b18_object_class_program_activity_1'
+
+not_req_dict = {
+        "submission_id":fuzzy.FuzzyInteger(9999),
+        "object_class":fuzzy.FuzzyInteger(1000,9999),
```

Comment:
I'm very confused by the use of `fuzzy.FuzzyInteger` here. Are you wanting them to be random values? If so, you probably want to use `randint` instead; `FuzzyInteger` is a class. However, if you're looking for random values, why specify them at all? FactoryBoy'll generate random values for you.